manifests/v/vassalengine/VASSAL/3.6.1/vassalengine.VASSAL.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: vassalengine.VASSAL
+PackageVersion: 3.6.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/vassalengine/vassal/releases/download/3.6.1/VASSAL-3.6.1-windows-aarch64.exe
+  InstallerSha256: FC9B60EE9B319D0B10C057427E88C701B7F680A4C58804164E38744210061130
+- Architecture: x86
+  InstallerUrl: https://github.com/vassalengine/vassal/releases/download/3.6.1/VASSAL-3.6.1-windows-x86_32.exe
+  InstallerSha256: AD73D8078FE063F626633250027DDA4AC2647BFF580E90FF86BC4C70BFCB0D35
```

```suggestion
- Architecture: arm64
  InstallerUrl: https://github.com/vassalengine/vassal/releases/download/3.6.1/VASSAL-3.6.1-windows-aarch64.exe
  InstallerSha256: FC9B60EE9B319D0B10C057427E88C701B7F680A4C58804164E38744210061130
- Architecture: x86
  InstallerUrl: https://github.com/vassalengine/vassal/releases/download/3.6.1/VASSAL-3.6.1-windows-x86_32.exe
  InstallerSha256: AD73D8078FE063F626633250027DDA4AC2647BFF580E90FF86BC4C70BFCB0D35
- Architecture: x64
  InstallerUrl: https://github.com/vassalengine/vassal/releases/download/3.6.1/VASSAL-3.6.1-windows-x86_64.exe
  InstallerSha256: 700ECC173F7D24B40932DA88C6C438EB9A920C3F42DCB7644A11D1DA547EF1A2
```

Use arm64 installer and add x64 installer.